projects/js-packages/components/components/donut-meter/style.scss

```diff
@@ -0,0 +1,14 @@
+.donut-meter {
+  .donut-meter-segment {
+    stroke: var(--jp-green);
+  }
+  &.is-info .donut-meter-segment {
+    stroke: var(--jp-green);
```

Comment:
Would this color be `blue` (--jp-blue-20 or --jp-blue-40) according to the annotation of prop `type`?

Reply:
Ah, let's go and remove `is-info` for now since we're not using it anywhere.